### PR TITLE
adding ted to bid tracker overlay

### PR DESCRIPTION
**Pull request by SidelineCory24**
<strike>Elizabeth is working on setting up the BE so the TED can be hooked into.</strike>
Setup is complete.

### CODE REVIEW

**Comment by elizabeth-jimenez** [comment]
Blocker: this [PR](https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/1510), just fyi

File: src/Components/BidTracker/OverlayAlert/HandshakeRegisterAlert/HandshakeRegisterAlert.jsx [hunk 1]
```diff
@@ -25,10 +24,7 @@ class HandshakeRegisterAlert extends Component {
     const post = getPostName(get(position, 'post'), NO_POST);
     const skillCode = get(position, 'skill') || NO_SKILL;
     const grade = get(position, 'grade') || NO_GRADE;
-    // saving ted for Cory
-    const ted = formatDate('2020-07-02T05:00:00Z');
-    // const ted = position.bid.ted ? formatDate(position.bid.ted) : NO_TOUR_END_DATE;
-    // modify line 6: import NO_TOUR_END_DATE from SystemMessages
+    const ted = formatDate(get(bid, 'position_info.ted')) || NO_TOUR_END_DATE;
```
Comment (SidelineCory24) [inline src/Components/BidTracker/OverlayAlert/HandshakeRegisterAlert/HandshakeRegisterAlert.jsx:27]: I was originally hooking into `position_info.current_assignment.estimated_end_date` 😅

**Review by elizabeth-jimenez — APPROVED** [review]
🍰 🍬 🥮 👍

**Comment by elizabeth-jimenez** [comment]
Looks good, just tests!

**Comment by elizabeth-jimenez** [comment]
> Looks good, just tests!

Rip i tried to cancel the previous test so only the last one runs, but i think i may have cancelled them all ⚰️  should be ok 🤞

**Pull request merged**